softwareComponents/simplesimServerLib/include/simplesim_server.hpp

```diff
@@ -0,0 +1,69 @@
+#pragma once
+
+#include <cassert>
+#include <filesystem>
+#include <ostream>
+
+#include <dimcli/cli.h>
+
+// RoFI headers
+#include <configuration/rofibot.hpp>
+#include <configuration/serialization.hpp>
+#include <configuration/universalModule.hpp>
+#include <simplesim/packet_filters/py_filter.hpp>
+
+
+namespace rofi::simplesim
+{
+enum class ConfigurationFormat
+{
+    Old = -1,
```

Comment:
Proč -1 a 0? Je za tím nějaká logika, kterou nevidiím?

Reply:
Je to proto, aby Json začínal na 0 - takže defaultní hodnota je 0 a časem by se to mohlo oddělat, tak aby se najednou neměnila hodnota ostatním

Reply:
A nemůžu mít opačné pořadí? JSON = 0, OLD = 1. Ale ono je to asi jedno.